src/aioswitcher/api/__init__.py

```diff
@@ -153,6 +153,24 @@ async def disconnect(self) -> None:
             logger.info("switcher device not connected")
         self._connected = False
 
+    async def _send_packet(self, packet_id: str, packet: str) -> bytes:
```

Comment:
If we only use the _packet_id_ to log a debug message, let's log it before invoking __send_packet_.

Reply:
The idea was to decrease file size.. If I do that it will increase this line again and again.

Reply:
@TomerFi  what do you think?

Reply:
I still think it should be outside the function. But it's not a deal breaker for me. We can go this way too.